src/components/ComponentLocking/AblyPoweredInput.tsx

```diff
@@ -21,20 +22,34 @@ export const AblyPoweredInput: React.FC<AblyPoweredInputProps> = ({
   const self = useSelf(space);
   const [value, setValue] = useLiveValue(spaceName, name, self);
 
-  const locked = Boolean(lockHolder);
+  // region Workaround to react on local lock release
+  const [unlocked, setUnlocked] = useState(false);
```

Comment:
mark all the code that just workaround for lock issue (without this it's not nice to test)